ci-operator/step-registry/rosa-regional-platform/teardown/rosa-regional-platform-teardown-commands.sh

```
@@ -18,5 +18,9 @@ git clone https://github.com/openshift-online/rosa-regional-platform.git "${WORK
 cd "${WORK_DIR}/platform"
 git checkout "${CLONE_REF}"
 
+# Generate AWS profiles from mounted credentials (idempotent, no-op if already set).
+# Backwards-compatible: older refs of rosa-regional-platform don't have this script.
+[[ -f ci/setup-aws-profiles.sh ]] && source ci/setup-aws-profiles.sh
```

Shellcheck (0.11.0)

[info] Not following: ci/setup-aws-profiles.sh was not specified as input (see shellcheck -x).

(SC1091)